src/templates/finops-hub/modules/storage.bicep

```diff
@@ -76,9 +90,181 @@ resource storageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' = {
   tags: union(tags, contains(tagsByResource, 'Microsoft.Storage/storageAccounts') ? tagsByResource['Microsoft.Storage/storageAccounts'] : {})
   properties: {
     supportsHttpsTrafficOnly: true
+    allowSharedKeyAccess: true
     isHnsEnabled: true
     minimumTlsVersion: 'TLS1_2'
     allowBlobPublicAccess: false
+    publicNetworkAccess: 'Enabled'
+    networkAcls: {
+      bypass: 'AzureServices'
+      defaultAction: enablePublicAccess ? 'Allow' : 'Deny'
+    }
+  }
+}
+
+resource scriptStorageAccount 'Microsoft.Storage/storageAccounts@2022-09-01' =  {
+  name: scriptStorageAccountName
+  location: location
+  sku: {
+    name: 'Standard_LRS' //sku
```

Comment:
Is this the cheapest storage account option?

How likely is it that a customer might have a policy that blocks this SKU? We can wait for feedback, but just wondering if it'd make sense to use the same SKU as the main storage account to avoid that.

Reply:
We can't use the primary storage account if it's premium storage because it doesn't offer an azure files endpoint.
https://learn.microsoft.com/en-us/azure/azure-resource-manager/templates/deployment-script-template#use-existing-storage-account